pages/1.11/release-notes/1.11.1/index.md

```diff
@@ -0,0 +1,111 @@
+- Pods now support persistent volumes. [View the documentation](/1.11/deploying-services/pods).[beta type="inline" size="small" /]
+
+**Note:** Because these storage features are beta in 1.11, they must be explicitly enabled. Beta features are not recommended for production usage, but are a good indication of the direction the project is headed.
+
+### Updated DC/OS Data Services
+- TLS encryption for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS is now supported.
+- Fault domain awareness for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic and DC/OS HDFS. Use fault domain awareness to make your services highly available and to allow for increased capacity when needed.
+- New API endpoint to pause a node for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS. Use this endpoint to relaunch a node in an idle command state for debugging purposes.
+- New DC/OS Kafka ZooKeeper service. [View the documentation](/services/kafka-zookeeper).
+- You can now select a DC/OS data service version from a dropdown menu in the DC/OS UI.
+- Improved scalability for all DC/OS data services.
+
+## <a name="issues-fixed"></a>Improvements and Issues Fixed in DC/OS 1.11.1 Release
+
+- INFINITY-3331 - Fixed cleaning up other framework's volumes.
+- DCOS_OSS-2292 - Fixed crashes on empty lines for DC/OS CLI.
+- DCOS_OSS-2255 - Fixed dcos-installer-ui package failure to build.
+- DCOS_OSS-2247 - Fixed fatal error on checks that exceeded timeout on dcos-diagnostic service.
+- DCOS_OSS-2223 - Added encoders to support package registry. Fixed comsos to initialize empty objects for optional json elements from json schema.
+- DCOS_OSS-2210 - Ensured that cache timestamps are well-formed.
+- DCOS_OSS-2184 - Removed an invalid dns record for marathon.mesos.
+- DCOS_OSS-2087 - Improved readability on user facing messages during service uninstallation.
+- DCOS_OSS-1759 - Updated package-manager.yaml to fix the schema error in packaage management API.
+- DCOS-21305 - Added a feature on DC/OS to support the minimal requested version when installing universe packages(cannot install a package which requires 1.11 on 1.10).
+- DCOS-21337 - Added linearBackoff retry to the Mesos stream.
+- DCOS-21266 - Fixed file navigation in the UI.
+- DCOS-21128 - Fixed uninstalling failing service breaks the services tab.
+- DCOS-20680 - Phrased the license violation message on banners.
+- DCOS-19648 - Added placement constraint validator to make sure the operator/ field combo is unique.
+- PRODUCT-288 - Enabled persistent volumes for PODs.
+- PRODUCT-224 - Storage service support in Marathon.
+- PRODUCT-893 - API endpoint to determine if DC/OS is OSS or EE.
+- INFINITY-3358 - Implemented UI with region picker.
+
+## <a name="notable-changes"></a>Notable Changes in DC/OS 1.11.1 Release
```

Comment:
How did we decide which tickets went in the notable changes section and which went into the Improvements and issues fixed section? Some of the issues below look like fixed issues.

Reply:
Also this 1.11.1 section should be at the top with the other 1.11.1 section. General 1.11 features should be underneath the new info.

Reply:
Quoting myself from elsewhere:

> ‘Notable changes’ vs ‘Fixed and improved’
> Great point. My rough thinking here is that if the content of the item does not directly communicate what has been "improved" or "fixed" then it's not suitable for the "Fixed and improved" section, but should go into a more neutral section.

> Or, in other words, for every point in "Fixed and improved" I want to know what about the change has fixed or improved something, and it's not really up for interpretation.

> These items should mainly be bugfixes, performance improvements, resilience improvements, UX improvements, etc.

> The following are examples for notable changes (changes of significance, but the kind of the significance is not communicated, and depends on the point of view):
"Platform: the Java Developer Kit was upgraded to JDK 8u152."
"DC/OS is now compatible with Docker version 17.05.0"
"DC/OS is now compatible with RHEL 7.4."